lc0/src/mcts/search.cc

```diff
@@ -216,6 +219,9 @@ void Search::Worker() {
         for (Node* n : node->Children()) {
           float p = computation.GetPVal(idx_in_computation,
                                         n->GetMove().as_nn_index());
+          if(p > 0.0f){
```

Comment:
Maybe
```cpp
  if (p != 1.0f && p > 0.0f) ...
```
?
I think it would save a tiny bit of time in the default case by not calling pow.

Reply:
done